addons/point_of_sale/views/account_statement_view.xml

```diff
@@ -19,16 +19,4 @@
                 </xpath>
             </field>
         </record>
-        <record id="view_account_bank_journal_form_inherited_pos" model="ir.ui.view">
-            <field name="name">account.bank.journal.form.inherit.pos</field>
-            <field name="model">account.journal</field>
-            <field name="inherit_id" ref="account.view_account_bank_journal_form"/>
-            <field name="arch" type="xml">
-                <xpath expr="//div[@name='advanced_info']" position="inside">
-                    <group string="Point of Sale">
-                        <field name="journal_user"/>
```

Comment:
same question here, we loose the journal_user field, is it okay? is it intended? Should we not set it on another view?

Reply:
it is still in `point_of_sale.view_account_journal_pos_user_form`

Reply:
good then :+1: